libs/ui/src/lib/drag-and-drop-file-input/drag-and-drop-file-input.component.html

```diff
@@ -0,0 +1,9 @@
+<div class="border-solid border border-black">
+  <ngx-dropzone multiple="false" (change)="selectFile($event)">
+    <div *ngIf="!fileName" class="text-black pl-4 py-4">
+      {{ placeholder | translate }}
+    </div>
+
+    <div *ngIf="fileName" class="text-black pl-4 py-4">{{ fileName }}</div>
+  </ngx-dropzone>
+</div>
```

Comment:
I would go for no border, smaller margin, and grey color.
I also prefere the attribute `translate` when possible

```suggestion
<div class="">
  <ngx-dropzone multiple="false" (change)="selectFile($event)">
    <div *ngIf="!fileName" class="text-gray-900 pl-2 py-2" translate="">
      {{ placeholder }}
    </div>
    <div *ngIf="fileName" class="text-gray-900 pl-2 py-2">{{ fileName }}</div>
  </ngx-dropzone>
</div>
```